### PR TITLE
ICU-21901 ICU demos README changes

### PR DESCRIPTION
**Pull request by sven-oly**
To improve the instructions. Also to improve the way that the demos are built and executed.

The BUILD steps need updating, too, in another PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,49 +99,47 @@ THE APPS
...
-     * Demonstrates the flexibility of the Calendar class.
+    Uses: Date/Time formatting, Calendar
 
-- Your contribution could be here too! Write: <icu4c@us.ibm.com>
+    * Demonstrates the flexibility of the Calendar class.
 
------------------
-(c) 1999, 2000 IBM, Inc. and others
+## Your contribution could be here too!
+Submit a suggestion at [ICU issue tracker](https://sites.google.com/site/icusite/bugs) and / or join the icu-support mailing list (see above.)
```
Comment (markusicu) [inline README.md:145]: At the end of the file please add
```
-----

### License

Please see [./LICENSE](./LICENSE)

> Copyright © 2016 and later Unicode, Inc. and others. All Rights Reserved.
Unicode and the Unicode Logo are registered trademarks
of Unicode, Inc. in the U.S. and other countries.
[Terms of Use and License](http://www.unicode.org/copyright.html)
```
@srl295 @yumaoka  This is from the icu-docs readme. In this file here, should we also retain a line for IBM? Maybe `Copyright © 1998-2016 IBM and Others.` just before the Unicode line?

File: README.md [hunk 2]
```diff
@@ -1,71 +1,66 @@
-Copyright (c) 1998-2009 IBM and Others.
-I C U  -  A P P S
------------------
+# I C U  -  A P P S
+#### Copyright © 1998-2009 IBM and Others.
```
Comment (markusicu) [inline README.md:2]: This line seems out of place. I suggest you remove it. (See my suggested addition at the end of the file.)

File: README.md [hunk 3]
```diff
@@ -1,71 +1,66 @@
...
 
- This package contains sample applications built using the ICU. For
-more information on the ICU:
 
- URL:   http://icu-project.org/
+This package contains sample applications built using the
+International Components for Unicode (ICU) C++ library ICU4C. For
+more information on the ICU, see [icu-project.org](http://icu-project.org/)
```
Comment (markusicu) [inline README.md:7]:
```suggestion
more information on ICU, see [icu-project.org](http://site.icu-project.org/)
```

File: README.md [hunk 4]
```diff
@@ -1,71 +1,66 @@
...
 
+If you have an application that is written using the ICU that you
```
Comment (markusicu) [inline README.md:9]:
```suggestion
If you have an application that is written using ICU that you
```

File: README.md [hunk 5]
```diff
@@ -1,71 +1,66 @@
...
+would like to contribute, join the ICU mailing list **icu-support** on [ICU contacts]
+(https://sites.google.com/site/icusite/contacts).
```
Comment (markusicu) [inline README.md:11]:
```suggestion
(http://site.icu-project.org/contacts).
```
We should not publish the Google Sites URLs.

File: README.md [hunk 6]
```diff
@@ -1,71 +1,66 @@
...
 
-  4. Type 'make' in the icuapps, or 'make install' if you wish.
+  4. Type 'make' in the icu-demos, or 'make install' if you wish.
 
- Example:
-
-  ICU:
-     cd ...somewhere/icu
+Example commands:
+````
```
Comment (markusicu) [inline README.md:35]: triple backticks, not quadruple backticks -- throughout

File: README.md (hunk 1 above)
Comment (markusicu) [inline README.md:145]:
```suggestion
Submit a suggestion at [ICU issue tracker](http://site.icu-project.org/bugs) and / or join the icu-support mailing list (see above.)
```

Comment (srl295) [reply README.md:145]: Probably safest to retain the one-liner…￼

Comment (srl295) [reply README.md:145]: Could easily be too much content here… It might be better to just say, "to contribute, see this link￼"

File: README.md [hunk 7]
```diff
@@ -1,71 +1,66 @@
-Copyright (c) 1998-2009 IBM and Others.
-I C U  -  A P P S
```
Comment (srl295) [inline README.md:2]:
```suggestion
# ICU Demos
```

**Comment by markusicu** [comment]
This PR targets the ICU 69rc BRS ticket, but ICU 69 was release a long time ago.

@sven-oly are the changes here still useful? If so, then please re-target them for an ICU 71 ticket and send this PR for review.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+- udata    *OBSOLETE - now part of udata in ICU*
+
+ - xlitomatic From: Steven R. Loomis
+                   <srl [at] icu-project.org>
+        
+    Uses: transliter-----
+
+### License
+
+Please see [./LICENSE](./LICENSE)ation
```
Comment (markusicu) [inline README.md:134]:
```suggestion
Please see [./LICENSE](./LICENSE)
```

File: README.md [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+-----
+
+### License
+
+Please see [./LICENSE](./LICENSE)
+Copyright © 2016 and later Unicode, Inc. and others. All Rights Reserved.
+Copyright © 1998-2016 IBM and Others.
+Unicode and the Unicode Logo are registered trademarks
+of Unicode, Inc. in the U.S. and other countries.
+[Terms of Use and License](http://www.unicode.org/copyright.html)
```
Comment (markusicu) [inline README.md:158]:
```suggestion
[Terms of Use and License](https://www.unicode.org/copyright.html)
```

File: README.md [hunk 10]
```diff
@@ -0,0 +1,158 @@
+# ICU Demos
+
+This package contains sample applications built using the
+International Components for Unicode (ICU) C++ library ICU4C. For
+more information on ICU, see [icu-project.org](http://site.icu-project.org/)
```
Comment (markusicu) [inline README.md:5]:
```suggestion
more information on ICU, see [icu.unicode.org](https://icu.unicode.org/)
```

File: README.md [hunk 11]
```diff
@@ -0,0 +1,158 @@
...
+
+If you have an application that is written using ICU that you
+would like to contribute, join the ICU mailing list **icu-support** on [ICU contacts]
+(http://site.icu-project.org/contacts).
```
Comment (markusicu) [inline README.md:9]:
```suggestion
(https://icu.unicode.org/contacts).
```

File: README.md [hunk 12]
```diff
@@ -0,0 +1,158 @@
...
+
+Note: [The "iucsamples" directory](https://github.com/unicode-org/icu-demos/tree/master/iucsamples) contains
+sample applications for the (International Unicode Conference (IUC) 43)
```
Comment (markusicu) [inline README.md:12]:
```suggestion
sample applications for the (Internationalization & Unicode Conference (IUC) 43)
```

File: README.md [hunk 13]
```diff
@@ -0,0 +1,158 @@
...
+[https://www.unicodeconference.org/iuc43/Conference_Program.pdf].  Each sample has its own
+build instructions.
+
+# Building ICU Demos
+
+## UNIX (or other command line platforms):
+
+  1. Build and install the ICU ('make install'). Make note of the 
+      prefix used to build the ICU, which can be set with the
+     '--prefix=' option to it's ./configure. The example directory /var/local/ is used in the samples below.
```
Comment (markusicu) [inline README.md:22]:
```suggestion
  1. Build and install ICU ('make install'). Make note of the 
      prefix used to build ICU, which can be set with the
     '--prefix=' option to its ./configure. The example directory /var/local/ is used in the samples below.
```

File: README.md [hunk 14]
```diff
@@ -0,0 +1,158 @@
...
+
+  2. icu-config doesn't need to be on your PATH - just make sure the same --prefix is used as that which built ICU.
+  
+  3. Run the ./configure script in the icu-demos directory.  You will need
+     to supply the same '--prefix=XXX' argument that was passed to the
+     ICU.
```
Comment (markusicu) [inline README.md:28]:
```suggestion
  3. Run the ./configure script in the icu-demos directory.  You will need
     to supply the same '--prefix=XXX' argument that was passed to ICU.
```

File: README.md [hunk 15]
```diff
@@ -0,0 +1,158 @@
...
+     ./configure --prefix=/var/local
+     make
+     make install                    (optional)
+```
+
+## Win32 Instructions
+  1. Build ICU in an 'icu' directory
+  2. Build icu-demos in an 'icuicu-demos' directory at the same level as 'icu'
+
+# RUNNING CGIS
```
Comment (markusicu) [inline README.md:50]:
```suggestion
# Running CGIs
```

File: README.md [hunk 16]
```diff
@@ -0,0 +1,158 @@
...
+           Uses : Conversion
+
+   This program will convert data file(s) from one encoding to another
+   via Unicode.
+   
+   As of ICU 1.9, this program is now part of the main ICU build. Though
+   it is still not supported it is more convenient there.
+
+- usort    From : Steven R. Loomis
+                  <srl [at] icu-project.org>
```
Comment (markusicu) [inline README.md:85]: @srl295 Do you want us to update your email address in this readme?

File: README.md [hunk 17]
```diff
@@ -0,0 +1,158 @@
...
+
+- locexp   From : Steven R. Loomis
+                  <srl [at] icu-project.org>
+
+    Uses : Conversion, Collation, Chartypes, Formatting,
+           Resource Bundles,  ...
+             uconv & usort [above]
+
+    * Shows off a lot of things. go to the ICU homepage and
+       look for the Locale Explorer.
```
Comment (markusicu) [inline README.md:105]:
```suggestion
    * Shows off a lot of things.
```
No direct link currently.

File: README.md [hunk 18]
```diff
@@ -0,0 +1,158 @@
...
+    * A way to look at the version and info headers on a udata
+      file
+
+
+- udata    *OBSOLETE - now part of udata in ICU*
+
+ - xlitomatic From: Steven R. Loomis
+                   <srl [at] icu-project.org>
+        
+    Uses: transliter-----
```
Comment (markusicu) [inline README.md:130]:
```suggestion
    Uses: Transliterator
```

File: README.md [hunk 19]
```diff
@@ -0,0 +1,158 @@
...
+
+### License
+
+Please see [./LICENSE](./LICENSE)ation
+      Depends on: locexp/util, etc.
+
+    * Transliterates HTML files using any transliterator. C++.
+ 
+- calexpo   From: Steven R. Loomis        [NOT CHECKED IN]
```
Comment (markusicu) [inline README.md:139]: @srl295 "NOT CHECKED IN", really? Why do we document something that we don't have?

File: README.md [hunk 20]
```diff
@@ -0,0 +1,158 @@
...
+                  <srl [at] icu-project.org>
+
+    Uses: Date/Time formatting, Calendar
+
+    * Demonstrates the flexibility of the Calendar class.
+
+## Your contribution could be here too!
+Submit a suggestion at [ICU issue tracker](http://site.icu-project.org/bugs) and / or join the icu-support mailing list (see above.)
```
Comment (markusicu) [inline README.md:147]:
```suggestion
Submit a suggestion at [ICU issue tracker](https://icu.unicode.org/bugs) and / or join the icu-support mailing list (see above.)
```

**Comment by sven-oly** [comment]
I am abandoning this PR, to be repositioned for a new ICU version.

**Pull request closed without merging**